e2e/src/usecases/WalletConnectV1.js

```diff
@@ -58,24 +52,10 @@ export default WalletConnect = () => {
   })
 
   beforeEach(async () => {
-    // Sleep a few seconds for runs in ci
+    // wait for any banners to disappear
     await sleep(3 * 1000)
   })
 
-  // Used to prevent a failing test spec form failing the entire test suite
-  afterEach(async () => {
```

Comment:
i think the components don't exist in the WC components anymore, they were from the full screen implementation